doc/ClassicalMaximals.bib

```diff
@@ -90,3 +90,9 @@ @article {MR11
        URL = {https://doi.org/10.1016/j.jsc.2010.09.003},
 }
 
+@article {T87,
+    AUTHOR = {Taylor, Donald E.},
+     TITLE = {Conjugacy of subgroups of the general linear group},
+      YEAR = {1987},
```

Comment:
```suggestion
      JOURNAL = {The Cayley Bulletin},
      VOLUME = {3},
      YEAR = {1987},
      URL = {https://www.maths.usyd.edu.au/u/don/papers/genAC.pdf},
```
It is still not nice because a DOI and some kind of long term storage would be preferable. Perhaps Don would be willing to upload it to arxiv.org or zenodo.org. I've now emailed Don about this.

Reply:
Don replied, he likes the idea of putting this on the arxiv, and even dug out the tex code for that. I expect it to appear there soon :-)